bin/fmt

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# DOC: Format terraform, python, and shell script files.
+
+if which -s terraform; then
+  echo "Formatting Terraform files"
+  terraform fmt -recursive -write
+else
+  echo "Terraform not found. Skipping."
```

Comment:
**suggestion (blocking):** should exit with an error code if it can't perform its function